src/test/java/seedu/address/logic/commands/DownloadAllNotesCommandTest.java

```diff
@@ -0,0 +1,24 @@
+package seedu.address.logic.commands;
+
+
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandFailure;
+import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
+
+import org.junit.*;
+import org.junit.rules.*;
+
```

Comment:
Here you imported the entire library again, for multiple libraries.  Are you sure you require all functions of all these libraries?